pkg/exchange/okex/convert.go

```diff
@@ -70,19 +64,19 @@ var CandleChannels = []string{
 	"candle30m", "candle15m", "candle5m", "candle3m", "candle1m",
 }
 
-func convertIntervalToCandle(interval types.Interval) string {
+func convertIntervalToCandle(interval types.Interval) WebSocketChannelType {
```

Comment:
Why haven't you defined the variables candle1h, candle2h, and so on?

Reply:
Actually  I defined in websocket.go